src/components/Dropdown/Dropdown.tsx

```diff
@@ -395,10 +394,13 @@ const Dropdown = ({
   const searchInputRef = useRef<HTMLInputElement>(null);
   const [searchCharacterCount, setSearchCharacterCount] = useState<number>(0);
   const [filteredOptions, setFilteredOptions] = useState<OptionProps[]>([]);
+  const stringifyOptions = JSON.stringify(options);
```

Comment:
this variable name sounds like a function - `stringifiedOptions` is probably better